scss/_variables.scss

```diff
@@ -467,11 +467,11 @@ $body-text-align:           null !default;
 // Style anchor elements.
 
 $link-color:                              $black !default; // Boosted mod
-$link-color-dark:                         $white !default; // Boosted mod
+$link-dark-color:                         $white !default; // Boosted mod
```

Comment:
You can't change all those variable names without deprecating them first. Maybe rather for v5.3 or v6. Regarding the discrepancies between the `$link-*-dark`, `$table-*-dark`, etc. and on the other hand `$breadcrumb-dark*`, etc.

Reply:
Yep I forgot about the deprecating. Creating an issue for that in order to not forget!